src/Components/CreateImageWizardV2/utilities/checkRepositoriesAvailability.ts

```diff
@@ -0,0 +1,74 @@
+import { useMemo } from 'react';
+
+import { useFormApi } from '@data-driven-forms/react-form-renderer';
+
+import { releaseToVersion } from './releaseToVersion';
+
+import { useListRepositoriesQuery } from '../store/contentSourcesApi';
+
+/**
+ * This checks the list of the payload repositories against a list of repos freshly
+ * fetched from content source API and returns true whether there are some
+ * repositories that are no longer available in the Repositories service.
+ *
+ * (The payload repositories are comming from the useFormApi hook).
```

Comment:
This isn't true anymore since we dropped DDF, right?

Reply:
The part about the `useFormApi` hook? It was dropped in one of the other commits: https://github.com/RedHatInsights/image-builder-frontend/pull/1569/commits/ba35ddeeb9f651fd755c902e0ab0da8d5e12b7b5#diff-89a5b0134c0bbbe72c49676be29f95346ac6852a06ee652ee3fcaea64a58b7b6

I've tried to structure the PR in a way that the files are first copied over in one commit and then updated in another one to show the diff a bit better.

Reply:
Oh super, yeah that totally makes sense, I'm just reviewing sequentially and adding comments immediately. Awesome!